library/SimplePie/Sanitize.php

```diff
@@ -145,6 +146,25 @@ public function pass_file_data($file_class = 'SimplePie_File', $timeout = 10, $u
 		}
 	}
 
+	public function rename_attributes($tags = array())
```

Comment:
Maybe we could prefix the method name with `set` to make it clear it is just a setter. I know the old setters do not follow this convention but we could start switching to more modern names (and deprecate the old ones after #711).

Reply:
I agree, it would be better. Let's do it in another PR. I want to focus here on the rename method